src/runtime/policy/policy.ts

```diff
@@ -279,6 +290,65 @@ export class PolicyField {
       label: label === 'raw' ? '' : label,
     };
   }
+
+  getRestrictedFields(parentType: Type) {
```

Comment:
What would you say about a one line comment describing the intention of this method?

Reply:
Changed the name to `toSchemaFields` and added a one line comment.